CONFIG.md

```diff
@@ -18,6 +18,8 @@ PEP_PERSEO = {}
 STH = {}
 PERSEO = {}
 CYGNUS = {}
+LDAP = {}
```

Comment:
Maybe an example could be included below, as with other endpoints.

Reply:
described below